### PR TITLE
Add "Branch" row at PR templates

### PR DESCRIPTION
**Pull request by phansys**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by sstok** [comment]
Can you please rebase 👍

**Comment by phansys** [comment]
Rebased.

File: src/Command/PullRequest/PullRequestCreateCommand.php [hunk 1]
```diff
@@ -79,6 +79,7 @@ protected function configure()
 your local <comment>.gush.yml</comment> file like:
 <comment>
 table-pr:
+    branch: ['Branch', master]
```
Comment (sstok) [inline src/Command/PullRequest/PullRequestCreateCommand.php]: This is only an example, so no need to add it here.

Comment (phansys) [inline src/Command/PullRequest/PullRequestCreateCommand.php]: I know it's a command help, but IMO it's better to enforce this row to be present.

Comment (sstok) [inline src/Command/PullRequest/PullRequestCreateCommand.php]: Fair enough.

**Comment by sstok** [comment]
Thank you @phansys

**Pull request merged**